git-version-control.Rmd

```diff
@@ -280,6 +280,7 @@ user community can engage in the development of your package.
 
 1.  Merge upstream with origin's devel branch,
 
+        git checkout -b devel
```

Comment:
Instead of this, add a note below it saying something like if the default branch is not devel use: 

```
git branch -M devel
```